src/components/SelectedTapHours.js

```diff
@@ -14,7 +14,7 @@ const SelectedTapHours = ({ infoIsExpanded, selectedPlace }) => {
 
   useEffect(() => {
     // update hoursList, currentOrgHours, and isOpen when selectedPlace changes
-    if (selectedPlace.hours !== undefined) {
+    if (selectedPlace.hours && selectedPlace.hours.length === 7) {
       const hoursList = [];
 
       selectedPlace.hours.map((orgHours, index) => {
```

Comment:
This section expects hours to have 7 entries. The days of the week are 0 indexed with 0 as Sunday. It looks like most of the data for the food taps do not have 7 entries. Most of them have 5.